Sources/Datadog/DDTracer.swift

```diff
@@ -38,23 +40,39 @@ public class DDTracer: Tracer {
     }
 
     internal convenience init(tracingFeature: TracingFeature, tracerConfiguration: Configuration) {
+        let serviceName = tracerConfiguration.serviceName ?? tracingFeature.configuration.serviceName
         self.init(
             spanOutput: SpanFileOutput(
                 spanBuilder: SpanBuilder(
                     applicationVersion: tracingFeature.configuration.applicationVersion,
                     environment: tracingFeature.configuration.environment,
-                    serviceName: tracerConfiguration.serviceName ?? tracingFeature.configuration.serviceName,
+                    serviceName: serviceName,
                     userInfoProvider: tracingFeature.userInfoProvider,
                     networkConnectionInfoProvider: tracingFeature.networkConnectionInfoProvider,
                     carrierInfoProvider: tracingFeature.carrierInfoProvider
                 ),
                 fileWriter: tracingFeature.storage.writer
+            ),
+            logOutput: TracingToLoggingOutput(
+                loggingOutput: LogFileOutput(
+                    logBuilder: LogBuilder(
+                        applicationVersion: tracingFeature.configuration.applicationVersion,
+                        environment: tracingFeature.configuration.environment,
+                        serviceName: serviceName,
+                        loggerName: "trace",
+                        userInfoProvider: tracingFeature.userInfoProvider,
+                        networkConnectionInfoProvider: tracingFeature.networkConnectionInfoProvider,
+                        carrierInfoProvider: tracingFeature.carrierInfoProvider
+                    ),
+                    fileWriter: tracingFeature.loggingFeatureStorage.writer
+                )
```

Comment:
does that mean basically we have 2 different loggers working separately? but they share the same `fileWriter` through `tracing/loggingFeature`?

Reply:
Exactly 👌. No matter of how many loggers are created (by the user or internally), they all share the same `FileWriter` instantiated for the `LoggingFeature`.